paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -169,7 +174,12 @@ void CompareResult(const std::vector<PaddleTensor> &outputs,
         float *pdata_ref = ref_out.data<float>(&place, &ref_size);
         EXPECT_EQ(size, ref_size);
         for (size_t j = 0; j < size; ++j) {
-          CHECK_LE(std::abs(pdata_ref[j] - pdata[j]), FLAGS_accuracy);
+          if (std::abs(pdata_ref[j]) > 1) {
+            CHECK_LE(std::abs((pdata_ref[j] - pdata[j]) / pdata_ref[j]),
+                     FLAGS_accuracy);
+          } else {
+            CHECK_LE(std::abs(pdata_ref[j] - pdata[j]), FLAGS_accuracy);
+          }
```

Comment:
Could you create a function to do this? It seems duplicated codes.

Reply:
Sure, I'll do that